lib/frisk.js

```diff
@@ -2,29 +2,23 @@
 const _ = require('lodash');
 const validateUUID = require('uuid-validate');
 
+const requestParameterLocations = ['body', 'params', 'query'];
+
 const validateArray = (validator, array) => {
     return _.reduce(array, (isValid, element) => {
         return isValid && validator(element);
     }, true);
 };
 
 const isObject = (value) => {
-    if (_.isString(value)) {
-        try {
-            value = JSON.parse(value);
-        }
-        catch(err) {
-            return false;
-        }
-    }
     return _.isPlainObject(value);
 };
 
 const confirmType = (type, value) => {
     if(type === Frisk.types.integer) {
         return _.isInteger(_.toInteger(value));
     } else if (type === Frisk.types.number) {
-        return _.isNumber(_.toNumber(value));
+        return _.isFinite(_.toNumber(value));
```

Comment:
I believe `_.isNumber(_.toNumber(value))` will return `true` for any javascript value.